Conceptpower+Spring/src/main/java/edu/asu/conceptpower/bean/ConceptEditBean.java

```diff
@@ -0,0 +1,182 @@
+package edu.asu.conceptpower.bean;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.stereotype.Component;
+
+import edu.asu.conceptpower.core.ConceptType;
+import edu.asu.conceptpower.core.POS;
+
+@Component
+public class ConceptEditBean implements POS {
```

Comment:
Sorry implements is not required here so removed it. Just importing the package will do